src/plugins/telemetryTable/components/table.vue

```diff
@@ -847,6 +851,16 @@ export default {
 
             this.isShowingMarkedRowsOnly = false;
         },
+        boundsChanged(_bounds, isTick) {
+            if (isTick) {
+                return;
+            }
+
+            if (this.paused) {
+                // Unpause the table and reset flags
+                this.unpause(true);
```

Comment:
Interesting - `unpause` expect a parameter that says whether or not it's been triggered by a button. This says `true`, but it clearly isn't triggered by a button. We should change the parameter name to something more meaningful.